.cicd/scripts/srw_metric.sh

```diff
@@ -105,7 +105,7 @@ if [[ ${RUN_STAT_ANLY_OPT} == true ]]; then
     elif [[ -f Indy-Severe-Weather.tgz ]]; then
         tar xvfz Indy-Severe-Weather.tgz 
     else
-        wget https://noaa-ufs-srw-pds.s3.amazonaws.com/sample_cases/release-public-v2.1.0/Indy-Severe-Weather.tgz
+        wget https://noaa-ufs-srw-pds.s3.amazonaws.com/experiment-user-cases/release-public-v2.2.0/METplus-vx-sample/Indy-Severe-Weather.tgz
```

Comment:
While attempting to run this update without using staged data, the metric test failed.  This is because the directory structure of the `Indy-Severe-Weather.tgz` tar file on AWS is:

`Indy-Severe-Weather/metprd/PointStat`

rather than:

`Indy-Severe-Weather/metprd/point_stat`

In order to correct this, line 103 needs to be changed to:

`mkdir -p Indy-Severe-Weather/metprd/PointStat`

line 116 needs to be changed to:

`cp ${we2e_experiment_base_dir}/${we2e_test_name}/2019061500/metprd/PointStat/*.stat ${workspace}/Indy-Severe-Weather/metprd/PointStat/`

and line 129 to be changed to:

`stat_analysis -config parm/metplus/STATAnalysisConfig_skill_score -lookin ${workspace}/Indy-Severe-Weather/metprd/PointStat -v 2 -out ${platform,,}-${srw_compiler}-skill-score.txt`

With these modifications the script will successfully pull the necessary data from AWS, but the stat_analysis encounters the following issue, resulting in no skill score being calculated:

```
WARNING: 
WARNING: SSIndexJobInfo::compute_ss_index() -> the number of aggregated forecast and reference lines differ for 17 of the 17 terms. Rerun at verbosity level 3 (-v 3) for details.
WARNING: 
WARNING: 
WARNING: SSIndexJobInfo::compute_ss_index() -> cannot compute the skill score for 17 of the 17 terms. Rerun at verbosity level 3 (-v 3) for details.
WARNING: 
WARNING: 
WARNING: SSIndexJobInfo::compute_ss_index() -> skipping GO_INDEX since the ratio of valid terms 0/17 < 1!
```
Changing `https://noaa-ufs-srw-pds.s3.amazonaws.com/experiment-user-cases/release-public-v2.2.0/METplus-vx-sample/Indy-Severe-Weather.tgz` to `https://noaa-ufs-srw-pds.s3.amazonaws.com/experiment-user-cases/release-public-v2.1.0/METplus-vx-sample/Indy-Severe-Weather.tgz` will allow the metric test to successfully pull the data and run without issue.

```suggestion
        wget https://noaa-ufs-srw-pds.s3.amazonaws.com/experiment-user-cases/release-public-v2.1.0/METplus-vx-sample/Indy-Severe-Weather.tgz
```